tests/feature_activation/test_feature_service.py

```diff
@@ -445,10 +394,8 @@ def test_get_state_from_active(storage: TransactionStorage, block_height: int) -
 
 
 @pytest.mark.parametrize('block_height', [16, 17, 18, 19])
-def test_caching_mechanism(storage: TransactionStorage, block_height: int) -> None:
```

Comment:
This test was removed because in part 5 the caching mechanism is removed, as it's not necessary anymore.